mkdocs/docs/tutorials/load-test-grpc/community/knative/loadtest.md

```diff
@@ -0,0 +1,80 @@
+---
+template: main.html
+---
+
+# Load test a Knative gRPC service
+
+!!! tip "Load test a Knative gRPC Service and validate SLOs"
+    Use an [Iter8 experiment](../../../../getting-started/concepts.md#what-is-an-iter8-experiment) to load test a [Knative](https://knative.dev/) gRPC service and validate latency and error-related service level objectives (SLOs).
+
+    **Use-case:** Continuous delivery (CD) of gRPC services is a motivating use-case for this experiment. If the gRPC service satisfies the SLOs specified in the experiment, it may be safely rolled out (for example, from a test environment to a production environment).
+
+    This experiment is illustrated in the figure below.
+
+    ![HTTP load test with SLOs](../../images/grpc-overview.png)
+
+
+???+ note "Before you begin"
+    1. [Complete the load test with SLO validation tutorial for Knative HTTP service](../../../load-test-http/community/knative/loadtest.md).
+    2. Update the Knative HTTP service used in the above tutorial to a Knative gRPC service as follows.
+    ```shell
+    kn service update hello \
+    --image docker.io/grpc/java-example-hostname:latest \
+    --port 50051 \
+    --revision-name=grpc
+    ```
```

Comment:
Is it necessary to run the HTTP test first? Would it be clearer if the instructions were:
1. Install Iter8
2. Install Knative and deploy your first Knative service
3. modify the service to be a gRPC service as follows:

Reply:
Agreed. Let me fix... and push again.